src/components/RunGraphSettings.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <p-pop-over
+    class="run-graph-settings"
+    auto-close
+    :placement="placement"
+  >
+    <template #target="{ toggle }">
+      <p-button
+        aria-label="Run Graph Options"
+        icon="CogIcon"
+        flat
+        @click="toggle"
+      />
+    </template>
+
+    <p-overflow-menu class="run-graph-settings__menu">
+      <p-label label="View">
```

Comment:
Nbd for now, but I do feel this distinction is tough to groc. The x access can be time bound or like.. "linear"? then the y axis or sorting can be arranged differently. Probably just semantics, maybe a simple flat list of layouts is better, not totally sure off the cuff.

Reply:
Yeah idk here. Technically the "view" is how the x axis is rendered and the "layout" is how the y axis is rendered. And it makes sense to mix and match both. But I'm also not 100% on this UX for controlling it. Important part is we can control it haha.